.github/workflows/release-pr.yaml

```diff
@@ -83,14 +84,15 @@ jobs:
     strategy:
       matrix:
         kubernetes-version:
-        - "1.23.13"
-        - "1.24.7"
-        - "1.25.9"
-        - "1.26.4"
-        - "1.27.1"
+          - "1.23.13"
+          - "1.24.7"
+          - "1.25.9"
+          - "1.26.4"
+          - "1.27.1"
```

Comment:
Why the heck is (what I presume is) a YAML auto-formatter using the indented array item syntax variant for this but then changing `chart-name` to the same indent as the parent field syntax variant.

YAML :frowning_face: